docs/usage/installation.md

```diff
@@ -1,17 +1,56 @@
 # Installation
 
-The `deltalake` project can be installed via Pip for Python or Cargo for Rust.
+The `deltalake` project can be installed via pip for Python or Cargo for Rust.
 
-## Using Pip
+## Install Delta Lake for Python
+
+With pip:
 
 ``` bash
 pip install deltalake
 ```
 
-NOTE: official binary wheels are linked against openssl statically for remote objection store communication. Please file Github issue to request for critical openssl upgrade.
+With Conda:
+
+```bash
+conda install -c conda-forge deltalake
+```
+
+## Install Delta Lake for Rust
 
-## Using Cargo
+With Cargo:
 
 ``` bash
 cargo add deltalake
-```
\ No newline at end of file
+```
+
+## Run Delta Lake and pandas in a Jupyter Notebook
+
+You can easily run Delta Lake and pandas in a Jupyter notebook.
+
+Create an environment file with the dependencies as follows:
+
+```yaml
+name: deltalake-minimal
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - python=3.11
+  - ipykernel
+  - pandas
```

Comment:
Let's add or replace it with Polars :)

Reply:
@ion-elgreco - sure, I added Polars too!